app/renderer/components/tabs/content/audioTabIcon.js

```diff
@@ -56,6 +58,39 @@ class AudioTabIcon extends React.Component {
     return props
   }
 
+  componentDidMount (props) {
+    this.transitionIfRequired()
+  }
+
+  componentDidUpdate (prevProps) {
+    this.transitionIfRequired(prevProps)
+  }
+
+  transitionIfRequired (prevProps) {
+    // transition only if we have an element and we're not pinned
+    if (!this.element || this.props.isPinned) {
+      return
+    }
+    // transition only if audio state changes
+    const hasChanged = !prevProps || this.props.showAudioIcon !== prevProps.showAudioIcon
+    // TODO: if we want to animate the unmounting of the component (when audio is stopped),
+    //       then we should use https://github.com/reactjs/react-transition-group
+    if (!hasChanged || !this.props.showAudioIcon) {
+      return
+    }
+    // TODO: measure element width if this ever becomes dynamic since it's not great to specify complex size logic in two places
+    //       ...or animate a child element using transform: translateX in order to achieve the slide-in
+    const frames = widthIncreaseElementKeyframes(0, globalStyles.spacing.iconSize)
```

Comment:
`frames` is used across several areas to reference the group of frames we have and can lead to confusion. Please change variable name to something more explicit like `elementKeyframes` or something you think fits better